docs/core/globals.md

```diff
@@ -51,3 +51,15 @@ global. This same interface is also exposed if requiring with CommonJS
 | ----------                   | -------------                                |
 | AFRAME                       | The object described above.                  |
 | hasNativeWebVRImplementation | Whether the client has native WebVR support. |
+
+## Running `AFRAME` in Node.js
+
+It is possible to run A-Frame in Node.js to get access to its globals. The only catch is you need to supply a browser `window` mock, since Node.js lacks a `window` object. A-Frame is tested with [jsom](https://github.com/tmpvar/jsdom), although any should work.
```

Comment:
Replace `you` with `we`

can remove the common before `since Node.js`

typo `jsom`

replace `any` with maybe `any JavaScript-based browser implementation